features/step_definitions/registry.rb

```diff
@@ -397,3 +397,44 @@
   cb_name ||= :generation_number
   cb[cb_name] = deployment(name).generation_number(user: user, cached: false)
 end
+
+Given /^docker config for default image registry is saved$/ do
```

Comment:
@xiuwang the step name is not very clear where `docker config` is saved to, please be more specific when the step name.

Reply:
@pruan-rht Thanks your advice, I have addressed this comment, please help review again.